Introduce an option to configure minimum coverage percantages

kl4n4 opened the pull request:
This might be helpful to others as well since in our company we do not have a 100% coverage requirement. So we introduced the option to reconfigure these limits (the default still remains at 100).

Besides that I also updated the `release.yml` to match that from the `tangro-actions-template`

ankri commented on README.md:
```diff
@@ -59,14 +59,30 @@ steps:
+  - name: Collect Coverage
+    uses: tangro/actions-coverage@1.1.0
+    env:
+      COVERAGE: 94
```
I would have preferred for the configuration values to be [parameters](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepswith) but am fine with using environment variables.

kl4n4 replied:
that does make sense - my first approach was to use environment variables but I agree, parameters are a better fit for these settings.
So I changed the implementation accordingly.

ankri commented on README.md:
````diff
@@ -59,14 +59,30 @@ steps:
       GITHUB_CONTEXT: ${{ toJson(github) }}
 ```
 
+There is also an option to adapt the minimum coverage percentage limits that need to be achived for the build to complete successfully (Defaults to `100`). You can configure an overall minimum by setting the environment variable `COVERAGE` and/or setting specific minimums for the different coverage metrics: `COVERAGE_LINES`, `COVERAGE_STATEMENTS`, `COVERAGE_FUNCTIONS` or `COVERAGE_BRANCHES`.
````
```suggestion
There is also an option to adapt the minimum coverage percentage limits that need to be achieved for the build to complete successfully (Defaults to `100`). You can configure an overall minimum by setting the environment variable `COVERAGE` and/or setting specific minimums for the different coverage metrics: `COVERAGE_LINES`, `COVERAGE_STATEMENTS`, `COVERAGE_FUNCTIONS` or `COVERAGE_BRANCHES`.
```

kl4n4 commented:
Hi,

you are very welcome and thanks for these great actions!
I found them while searching for simple and easy to use github actions for Jest and all others were unnecessary complex or did not work at all. So thanks for putting them OS! They do work great for our projects as well (besides the fact that some legacy apps need a configurable coverage value😉)

I changed the configuration to parameters and fixed the typo in the README. :)
